source/main/assets/tutorial/fr/references.json

```diff
@@ -0,0 +1,33 @@
+[
+	{
+		"id": "Marx1971",
+		"type": "book",
+		"event-place": "Paris, France",
+		"language": "français",
+		"number-of-pages": "319",
+		"publisher": "Éditions sociales",
+		"publisher-place": "Paris, France",
+		"source": "Library Catalog - www.sudoc.abes.fr",
+		"title": "Le Capital: critique de l'économie politique",
+		"title-short": "Le Capital",
+		"author": [
+			{
+				"family": "Marx",
+				"given": "Karl"
+			}
+		],
+		"translator": [
+			{
+				"family": "Roy",
+				"given": "Joseph"
+			}
+		],
+		"issued": {
+			"date-parts": [
+				[
+					"1971"
+				]
+			]
+		}
+	}
+]
```

Comment:
Here we need a trailing newline (i.e.: end the file with an empty line)

Reply:
ok done :)